app/controllers/api/account.php

```diff
@@ -1421,7 +1421,7 @@
 
         $queries = Query::parseQueries($queries);
         $grouped = Query::groupByType($queries);
-        $limit = $grouped['limit'] ?? APP_LIMIT_COUNT;
+        $limit = $grouped['limit'] ?? (int) App::getEnv('_APP_LIMIT_COUNT', APP_LIMIT_COUNT);
```

Comment:
Instead of changing  it in every file, it might be worth considering changing it in one place here

https://github.com/appwrite/appwrite/blob/c8689d1d416716e74a47c95957f96d15ea23baf0/app/init.php#L87

```php
define("APP_LIMIT_COUNT", App::getEnv('_APP_LIMIT_COUNT', 5000));
```

My only issue with this approach is its not consistent with the way we define other constants.. 🤔

Reply:
Agreed!

That's why I've chosen the first approach.

Reply:
Let's wait for more feedback from the team, maybe we can make an exception for this or refactor all the constants to use `define`

Reply:
I'm not opposed to changing all to `define`, some other constants may need to be set at runtime in future

Reply:
I can create new PR changing all `const`s to `define`s